.github/workflows/publish.yaml

```diff
@@ -0,0 +1,36 @@
+# This workflow uses actions that are not certified by GitHub.
```

Comment:
Trenger vi denne disclaimer'n i vår fil?

Reply:
Vet ikke hvorfor den er der. Er sånn i de andre bibliotekene vi har i Flyt. Workflow actions for bibliotekene burde kanksje oppdateres i samme stil som tjenestene ellers?

Reply:
Jeg tipper noen har vært ute med ✂️ og limstift..